controllers/machineset_controller.go

```diff
@@ -534,7 +535,11 @@ func (r *MachineSetReconciler) MachineToMachineSets(o client.Object) []ctrl.Requ
 		}
 	}
 
-	mss := r.getMachineSetsForMachine(context.TODO(), m)
+	mss, err := r.getMachineSetsForMachine(context.TODO(), m)
```

Comment:
Can we define this context only once at the top of the function?

Reply:
Done